chia/daemon/server.py

```diff
@@ -656,18 +634,18 @@ def plot_queue_to_payload(self, plot_queue_item, send_full_log: bool) -> Dict[st
             item["log"] = plot_queue_item.get("log")
         return item
 
-    def prepare_plot_state_message(self, state: PlotEvent, id):
+    def prepare_plot_state_message(self, state: PlotEvent, plot_task_id):
```

Comment:
This seems unrelated - we should remove from this PR?

Reply:
Me and mariano did those, having a variable named id can screw up python stuff and can cause errors.

Reply:
Please do it in another PR; it may be a good idea, it's just wholly unrelated to this PR

Reply:
> Please do it in another PR; it may be a good idea, it's just wholly unrelated to this PR

ok, time for pain.

Reply:
> ok, time for pain.

Builds character.

Reply:
done